src/linear_algebra/dual.lean

```diff
@@ -78,7 +78,7 @@ variables [field K] [add_comm_group V] [vector_space K V]
 open vector_space module module.dual submodule linear_map cardinal function
 
 variables [de : decidable_eq ι]
-variables {B : ι → V} (h : is_basis K B)
+variables (B : ι → V) (h : is_basis K B)
```

Comment:
Could you explain why you made `B` explicit here? It seems like it could be inferred from `h` everywhere.

Reply:
Otherwise you see `to_dual _` in the pretty printer if the proof is not just a variable.

Reply:
Aha, I had never even thought about that! Could you add this to a comment on `to_dual` so if we ever get per-definition control of `pp.proofs`, we know that we can make `B` implicit again?

Reply:
If I have a long proof, I usually prefer to see `B`, not the proof.

Reply:
Hmm, I tend to leave `B` complicated and bundle the proof up into a nice definition :)